packages/paste-website/src/pages/primitives/combobox-primitive/index.mdx

```diff
@@ -154,12 +160,28 @@ getter or directly on the element itself. You can see this demonstrated below:
 This hook can be used to create custom autocomplete Combobox controls. These controls are useful when the
 customer needs to filter a list of available options, or provide a custom free form value to the input.
 
+<LivePreview scope={{useComboboxPrimitive, Box, Button, ChevronDownIcon, Label, Input, useUID}} noInline language="jsx">
+  {autocompleteExample}
+</LivePreview>
+
+#### Multiselect Combobox Example
+
 <LivePreview
-  scope={{useComboboxPrimitive, Box, Button, ChevronDownIcon, FormLabel, FormInput, useUID}}
+  scope={{
+    useComboboxPrimitive,
+    useMultiSelectPrimitive,
+    Box,
+    Button,
+    ChevronDownIcon,
+    Label,
+    Input,
+    useUID,
+    useUIDSeed,
+  }}
   noInline
   language="jsx"
 >
-  {autocompleteExample}
+  {multiSelectExample}
 </LivePreview>
 
 ### useComboboxPrimitive Arguments
```

Comment:
You just need to add the `useMultiselectPrimitive` arguements and props to the page.

I think I just copy and pasted them from downshift last time, and just tidied up any formatting.